app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -251,6 +250,7 @@ import com.duckduckgo.common.utils.extensions.html
 import com.duckduckgo.common.utils.extensions.websiteFromGeoLocationsApiOrigin
 import com.duckduckgo.common.utils.extractDomain
 import com.duckduckgo.common.utils.plugins.PluginPoint
+import com.duckduckgo.customtabs.impl.pixels.CustomTabPixelNames
```

Comment:
The `app` module ideally wouldn't be touching the `impl` of another module. 

Should these pixels actually belong to the `app` module since that's where they are invoked?